beacon_node/network/src/sync/network_context.rs

```diff
@@ -142,13 +151,41 @@ impl<T: BeaconChainTypes> SyncNetworkContext<T> {
             request_id: 1,
             blocks_by_root_requests: <_>::default(),
             blobs_by_root_requests: <_>::default(),
+            data_columns_by_root_requests: <_>::default(),
             range_blocks_and_blobs_requests: FnvHashMap::default(),
             network_beacon_processor,
             chain,
             log,
         }
     }
 
+    // TODO(das): epoch argument left here in case custody rotation is implemented
+    pub fn get_custodial_peers(&self, _epoch: Epoch, column_index: ColumnIndex) -> Vec<PeerId> {
+        let mut peer_ids = vec![];
+
+        for (peer_id, peer_info) in self.network_globals().peers.read().peers() {
+            if let Some(enr) = peer_info.enr() {
+                // TODO(das): do not hardcode `custody_subnet_count`
+                let custody_subnet_count = 2;
```

Comment:
Should be able to use `enr.custody_subnet_count()` for this now.

https://github.com/sigp/lighthouse/blob/d5e7e73abc64312d7a6f5ce3a54b0df455e2b243/beacon_node/lighthouse_network/src/discovery/enr.rs#L67